diff --git a/src/schema/schema.generated.ts b/src/schema/schema.generated.ts
index d90dd1d..3c9af6b 100644
--- a/src/schema/schema.generated.ts
+++ b/src/schema/schema.generated.ts
@@ -16,9 +16,11 @@ export type TypeRepresentation =
       type: "string";
     }
   | {
+      /** @deprecated since NDC Spec v0.1.2 - use sized numeric types instead */ // Manually added :(
       type: "number";
     }
   | {
+      /** @deprecated since NDC Spec v0.1.2 - use sized numeric types instead */ // Manually added :(
       type: "integer";
     }
   | {
@@ -714,9 +716,7 @@ export interface RowSet {
       }[]
     | null;
 }
-export interface RowFieldValue {
-  [k: string]: unknown;
-}
+export type RowFieldValue = unknown; // Manually corrected :(
 export interface MutationRequest {
   /**
    * The mutation operations to perform
