diff --git a/src/schema/schema.generated.ts b/src/schema/schema.generated.ts
index 372b5ee..dc8fed2 100644
--- a/src/schema/schema.generated.ts
+++ b/src/schema/schema.generated.ts
@@ -1101,9 +1101,7 @@ export interface RowSet {
    */
   groups?: Group[] | null;
 }
-export interface RowFieldValue {
-  [k: string]: unknown;
-}
+export type RowFieldValue = unknown; // Manually corrected :(
 export interface Group {
   /**
    * Values of dimensions which identify this group
